bandwidth/voice/__init__.py

```diff
@@ -1,7 +1,6 @@
-__all__ = [
-    'bxml',
+__all__ = [
```

Comment:
Is bxml meant to be removed from here?

Reply:
It is not, good catch. Fixed. I wonder why/how the tests still ran and passed without this...